automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspacesAzureApiSpec.scala

```diff
@@ -35,24 +35,23 @@ class AzureWorkspacesSpec extends AnyFlatSpec with Matchers with CleanUp {
   implicit val token: AuthToken = ownerAuthToken
 
   "Rawls" should "allow creation and deletion of azure workspaces" in {
-    assert(true)
-//    withTemporaryAzureBillingProject(azureManagedAppCoordinates) { projectName =>
-//      val workspaceName = generateWorkspaceName()
-//      Rawls.workspaces.create(
-//        projectName,
-//        workspaceName,
-//        Set.empty,
-//        Map(AttributeName.withDefaultNS("disableAutomaticAppCreation") -> AttributeBoolean(true))
-//      )
-//
-//      val response = workspaceResponse(Rawls.workspaces.getWorkspaceDetails(projectName, workspaceName))
-//      response.workspace.name should be(workspaceName)
-//      response.workspace.cloudPlatform should be(Some(WorkspaceCloudPlatform.Azure))
-//      response.workspace.workspaceType should be(Some(WorkspaceType.McWorkspace))
-//
-//      Rawls.workspaces.delete(projectName, workspaceName)
-//      assertNoAccessToWorkspace(projectName, workspaceName)
-//    }
+    withTemporaryAzureBillingProject(azureManagedAppCoordinates) { projectName =>
+      val workspaceName = generateWorkspaceName()
+      Rawls.workspaces.create(
+        projectName,
+        workspaceName,
+        Set.empty,
+        Map(AttributeName.withDefaultNS("disableAutomaticAppCreation") -> AttributeBoolean(true))
```

Comment:
@marctalbott will change to

`Map("disableAutomaticAppCreation" -> "true")`

Reply:
And will need to change this as well: https://github.com/broadinstitute/rawls/pull/2339/files#diff-518dd904cf9703f8db8998f3dd2eab4c5ef7b5488d59532eb194a391370e63e7R586